examples/provider.yaml

```diff
@@ -17,14 +17,3 @@ spec:
       name: alibaba-account-creds
       key: credentials
   region: cn-beijing
-
```

Comment:
Please see the comment above regarding the `ProviderConfig` version used in this file.

Reply:
Done.